pyproject.toml

```diff
@@ -35,6 +35,8 @@ orjson = "^3.9"
 pynumaflow = "~0.7.2"
 prometheus_client = "^0.18.0"
 structlog = "^24.1.0"
+numalogic-prometheus = { path = "numalogic_prometheus-0.8.0-py3-none-any.whl" }
```

Comment:
We need to change this. Can we publish first?

Reply:
Yes